Makefile

```diff
@@ -3,7 +3,7 @@
 test: unit integration
 
 unit:
-	./scripts/run-local-tests.sh
+	SKIP_COMPOSE_API_TESTS=true ginkgo --timeout 30m -r
 
 integration:
-	ginkgo -p --nodes=8 -r integration_tests
```

Comment:
Why was parallelism removed in this commit? Should it be part of 00390ce6ca41782a60ff938858cf6095cedd3bd5 where `-nodes` is changed to `2`?

Reply:
🙌  think it must've got muddled in a rebase dance